release: Adds changelog for 1.1.0 release (#406)
elastic/ecctl: docs/release_notes/v1.1.0.adoc

https://download.elastic.co/downloads/ecctl/1.1.0/ecctl_1.1.0_linux_amd64.tar.gz[ecctl_1.1.0_linux_amd64.tar.gz]

[float]
[id="{p}-release-notes-v1.1.0-whats-new"]
==== What's new

* *Authorization headers are now redacted when using `--verbose`.* The default behavior of the `--verbose` flag has been changed to redact the contents of the Authorization header in outgoing http requests. To disable this behavior, set `--verbose-credentials` or `verbose_credentials: true` in the configuration file.

* *New global flags.* Two new flags have been added:
+
--
`ecctl --verbose-credentials`:: When set, Authorization headers on the request/response trail will be displayed as plain text.
`ecctl --verbose-file <string>`:: When set, the verbose request/response trail will be written to the defined file.
--

* *New configuration options.* Two new configuration options have been added:
+
--
`verbose_credentials (bool)`:: When set, Authorization headers on the request/response trail will be displayed as plain text.
`verbose_file (string)`:: When set, the verbose request/response trail will be written to the defined file.
--

> * *Allocator list command now supports a `--size` flag.* With the addition of the new flag, it allows users to obtain more than the default per region allocator limit of 100.

Reviewer: ```suggestion
* *Allocator list command now supports a `--size` flag.* With the addition of the new flag, users are able to obtain more than the default per region allocator limit of 100.
```
